simpleflow/swf/mapper/exceptions.py

```diff
@@ -1,18 +1,11 @@
-# Copyright (c) 2013, Theo Crevon
```

Comment:
Looks like you removed this block in some files and added it in others? 🙂

Reply:
I'm changing my mind: I removed nothing in this PR, it was in #422 or #423. I see 8 added lines with "# Copyright", and 10 removed ones. So only one file where I actively removed the thing, and it's probably not worth investigating since we want to remove them all. I hope Greg/Théo won't take offence, worst case I'll restore later.

Thanks for the eagle eyes anyway, I didn't notice 😅